### DISCUSSION TITLE
Congo Mascot / Icon / Logo ?

### DISCUSSION THREAD

**Discussion by wolfspyre** [jpanther/congo#735] [General]
Wondering what the mascot / theme icon / logo for Congo should be.

Relevant, as it would be more intuitive from a documentation perspective to have a placeholder persona for the construct of "site owner / site author / site majordomo"   
in contrast to "contributors" which represent the cohort of individuals contributing content to a collaboratively authored site. 

I've got a few ideas... 

- Creatures with emoji: 
  - a gorilla  🦍
  - a parrot 🦜
  - a panther 🐆
- other naturey-things with emoji: 
  - palm 🌴
  - $tree 🌳
  - $mountain ⛰️
  - $snowyMountain🏔️
  - fuji🗻
  - national park🏞️ (this was a neat discovery)

These are easy reaches as they've got existing (mostly)ubiquitous iconography... but that's not really the only important factor.
Other concerns exist around idiomatic apropos, ex/inclusivity, and reasonable sensitivity to existing associations with icons in common use, ie we prolly shouldn't use... uhh... ` the chocolate cake monster` 💩 emoji.

There are a bunch of other ideas for mascots, or theme personas too... I'm just trying to see what peeps think 

I don't want to forcefully steer here... rather I want to encourage consensus/a decision/direction.

My inclination is is to go with something like a friendly gorilla and just start rolling with it... but I feel that a conversation has to have been started, and some time has passed before it's reasonable for me to start off in any direction sans input ;)

**Comment by rshmhrj** [comment]
I always thought the mascot was this guy:
<img width="246" alt="image" src="https://github.com/jpanther/congo/assets/750874/fe793be7-db0f-4814-b85a-0512ebfaf391">

🤷🏼
